src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_img2img.py

```diff
@@ -1012,3 +1013,73 @@ def denoising_value_valid(dnv):
             return (image,)
 
         return StableDiffusionXLPipelineOutput(images=image)
+
+    # Overrride to properly handle the loading and unloading of the additional text encoder.
+    def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]], **kwargs):
```

Comment:
We can use `# Copied from ...` statements here and elsewhere where we're verbatim copying methods. 

Example:

https://github.com/huggingface/diffusers/blob/aef11cbf667ce5175fdca2937a8ab7df385ebe49/src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py#L67

Reply:
`# Copied from diffusers.pipelines.stable_diffusion_xl.pipeline_stable_diffusion_xl.load_lora_weights`
Thanks, I just left `Copy from ...` statement in front of the functions.